packages/app/src/app/pages/Dashboard/Components/shared/StyledCard.tsx

```diff
@@ -12,9 +12,6 @@ export const StyledCard = styled(Card)<StyledCardProps>(({ dimmed }) => ({
   '&:hover': {
     backgroundColor: '#252525',
   },
-  '&:has(button:hover)': {
-    backgroundColor: '#1D1D1D',
-  },
```

Comment:
This was messing with the `Card` hover when it only contained one button. If we want to "reset" the card background when a (icon) button is hovered, we need to rethink this.